srtcore/congctl.cpp

```diff
@@ -492,9 +521,9 @@ class FileCC: public SrtCongestionControlBase
         if (CSeqNo::seqcmp(lossbegin, m_iLastDecSeq) > 0)
         {
             m_dLastDecPeriod = m_dPktSndPeriod;
-            m_dPktSndPeriod = ceil(m_dPktSndPeriod * 1.125);
+            m_dPktSndPeriod = ceil(m_dPktSndPeriod * 1.03);
 
-            m_iAvgNAKNum = (int)ceil(m_iAvgNAKNum * 0.875 + m_iNAKCount * 0.125);
+            m_iAvgNAKNum = (int)ceil(m_iAvgNAKNum * 0.97 + m_iNAKCount * 0.03);
```

Comment:
Shouldn't it be more configurable, as you already change it here? For example:
```
const double FILECC_LOSS_SHARE_FACTOR = 0.03;
(....)
m_dPktSndPeriod = ceil(m_dPktSndPeriod * (1 + FILECC_LOSS_SHARE_FACTOR));

m_iAvgNAKNum = (int)ceil(m_iAvgNAKNum * (1 - FILECC_LOSS_SHARE_FACTOR)
              + m_iNAKCount * FILECC_LOSS_SHARE_FACTOR);

Reply:
Made a local constant for now. There will be more changes to FileCC anyway.